.github/workflows/ci.yaml

```diff
@@ -63,6 +63,7 @@ jobs:
         run: |
           make test
           git status
+          cat api/v1alpha1/zz_generated.deepcopy.go
```

Comment:
Why add this?

Reply:
Thanks! This situation is `https://github.com/openkruise/rollouts/issues/99`.  For solving my question, Mr.zmberg make some tests in CI.
In commit `[ffa019e](https://github.com/openkruise/rollouts/pull/61/commits/ffa019e5c4b5013648124ce4dd283c8303110a7a)`, there are three files changed.

Reply:
It has been removed now, thanks!